### PR TITLE
nicotine-plus 1.4.1 (new formula)

### PR DESCRIPTION
**Pull request by ibrokemypie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ibrokemypie** [comment]
Been a week, any update on this? Anything I need to do?

File: formula/nicotine-plus.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class NicotinePlus < Formula
+  include Language::Python::Virtualenv
+
+  desc "is a graphical client for the SoulSeek peer-to-peer system."
```
Comment (bfontaine) [inline formula/nicotine-plus.rb]: Please don’t start the description with `is`: `"Graphical client for the SoulSeek peer-to-peer system"`.

File: formula/nicotine-plus.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  homepage "http://www.nicotine-plus.org/"
+  url "https://github.com/Nicotine-Plus/nicotine-plus/archive/1.4.1.tar.gz"
+  sha256 "1b38ef196d981e4eb96fa990cc463143289784f988f559c4400b1d461497b7d6"
+  head "https://github.com/Nicotine-Plus/nicotine-plus.git"
+  depends_on :python if MacOS.version <= :snow_leopard
```
Comment (bfontaine) [inline formula/nicotine-plus.rb]: Please leave a blank line above this one.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: formula/nicotine-plus.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  depends_on "gtk+"
+  depends_on "pygtk"
+  depends_on "geoip" => :optional
+  depends_on "miniupnpc" => :optional
```
Comment (bfontaine) [inline formula/nicotine-plus.rb]: Why would someone not build with these dependencies?

Comment (ibrokemypie) [inline formula/nicotine-plus.rb]: If they simply do not need them it reduces the bloat

Comment (ibrokemypie) [inline formula/nicotine-plus.rb]: would you like me to make them recommended or required?

File: formula/nicotine-plus.rb [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    venv = virtualenv_create(libexec)
+    venv.pip_install "mutagen"
+    venv.pip_install "miniupnpc" if build.with? "miniupnpc"
+    venv.pip_install "python-geoip" if build.with? "geoip"
+    venv.pip_install_and_link buildpath
+  end
+
+  test do
+    system "#{bin}/nicotine", "--version"
+    system "#{bin}/nicotine", "--help"
```
Comment (dunn) [inline formula/nicotine-plus.rb]: Is it possible to have the tests check more of the functionality?

Comment (ibrokemypie) [inline formula/nicotine-plus.rb]: no really sure what else I can test, similarly to the qmmp one, this one spawns a gtk window with any other options, so will always fail

**Comment by dunn** [comment]
The bottle isn't building since you added this as `formula/nicotine-plus.rb` instead of `Formula/nicotine-plus.rb` (notice the case difference).

Are you able to fix that on your end?

**Comment by ilovezfs** [comment]
@ibrokemypie I have fixed it for you. Just so we have a better idea of how this happened: is your local directory actually named `formula` not `Formula`?

**Pull request closed without merging**